src_to_mesh.cpp

```diff
@@ -99,7 +106,11 @@ int main(int argc, char** argv) {
 
 
   std::vector<double> rel_strengths(bin_counts.begin(), bin_counts.end());
-  for (auto& rs : rel_strengths) { rs /= sites_sampled; }
+  for (auto& rs : rel_strengths) { rs /= (sites_sampled-missed_bins); }
```

Comment:
I think we should be careful here. When accounting for source sites outside of the mesh, the meaning of the relative strengths reported by this program can be either:

  - relative to the original source (what we were reporting before)
  - relative to the source that lies in the mesh (what this change will report)
  
  I think it makes the most sense to do the former because you can always renormalize the source strengths to get the latter, but you can't go the other way. Thoughts?

Reply:
For my application this way works slightly better; this is the number that I want to have for both plotting and for GeN-Foam, so this way allows me to not have to normalize before plotting or in the translation into GeN-Foam. Maybe we could make this an option? Relative to the original source may be more intuitive so it could be the default

Reply:
If this CSV file is being processed in another way (another C++ program for example) maybe it does make sense to have these values in there. In that case I'd probably vote to include both fields in the output file.

Reply:
Yes, it will be a python script just doing some formatting work. The main reason I would like to have at least one of the fields is for plotting purposes, otherwise I would have to use python/excel (or similar) as a separate step before importing the csv into paraview. It's not impossible to do, just an extra step if we aren't worried about overflow or memory for the first C++ script

Reply:
Ah I see, if you're using the CSV file directly in paraview then I can see how avoiding an intermediate script would be nice. What I'll recommend then is writing both fields into the CSV file rather than one or the other.